example/heatEquation/src/heatEquation.cpp

```diff
@@ -163,15 +158,24 @@ auto main() -> int
     alpaka::memcpy(queue, uNextBufAcc, uCurrBufAcc, extent);
     alpaka::wait(queue);
 
+    auto* uCurrBufAccPtr = &uCurrBufAcc;
```

Comment:
Why is the `*` necessary here?

Reply:
It is not. It is just some emphasis that these variables are pointers.